config/zones/AD.yaml

```diff
@@ -1,14 +1,16 @@
 bounding_box:
-  - - 0.9064563390001013
-    - 41.92867747000005
-  - - 2.2650907800001505
-    - 43.14936167400003
+- - 0.9064563390001013
+  - 41.92867747000005
+- - 2.2650907800001505
+  - 43.14936167400003
 capacity:
   biomass: 5
   gas: 1.7
   hydro: 46
   solar: 0.231
 contributors:
-  - Manu1400
-  - autipial
+- Manu1400
+- autipial
 timezone: Europe/Andorra
+region: Europe
+country: AD
```

Comment:
Do we really need the country? It should always be the 2 first chars of the zoneKey.

I'm not against adding it I'm just wondering why you might need this?

Reply:
I forgot about those! 😅 Let's add the key then. I'm unsure about the other naming though but that we can look at later if need be.